### DISCUSSION TITLE
Custom Request?  Spoof webcam as screen region?

### DISCUSSION THREAD

**Discussion by phillypro** [tom-englert/RegionToShare#43] [Ideas]
I use an app that has limits on how many people can view your webcam
but no limits on how many people can view your screenshare

which is why i use this app

is it possible to directly grab the output of a webcam and use it as a screenshare region?

**Comment by tom-englert** [comment]
So any app that shows the cam output in a separate window should be able to do the job?

**Reply by phillypro** [reply]
kinda sorta
right now i use streamlabs OBS which has an output projector...
but of course therese a title bar .....so when i go to share screen it shows the entire window with title bar 
i use region to share to just select the part of the output without the title bar

but that loses the ability of being able to share an "app window" sitting in the background behind another window
now its sharing the "screen"

i wonder if its possible to crop an app window with another app or lock onto it or something.....it would just be awesome

**Reply by tom-englert** [reply]
Just build your own, so you have control over the borders
https://learn.microsoft.com/en-us/windows/uwp/audio-video-camera/simple-camera-preview-access
